### ISSUE TITLE
Gradle build fails on Windows

### ISSUE THREAD

**Issue by xatapult** [xproc/xproc.org#7]
When I try to run the Gradle build for this repo on my Windows computer it fails. Output:

```
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Data\Erik\work\xproc-3\xproc.org\build.gradle' line: 131

* What went wrong:
A problem occurred evaluating root project 'xproc.org'.
> Illegal character in opaque part at index 2: C:\Data\Erik\work\xproc-3\xproc.org\src\about.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.8/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 11s
```

Seems that somewhere a `file:/` is missing?

I think the only one that can fix this is @ndw , could you have a look?

**Comment by xatapult** [comment]
@ndw: Sorry, but the build still fails, in a *different* way. What I did after pulling your changes in  was a `gradle clean` followed by `gradle`. Here is the full output of both commands:

```
C:\Data\Erik\work\xproc-3\xproc.org>gradle clean

BUILD SUCCESSFUL in 1s
1 actionable task: 1 executed
C:\Data\Erik\work\xproc-3\xproc.org>gradle

> Task :src_about.html FAILED
file:/C:/Data/Erik/work/xproc-3/xproc.org/src/about.html:12:23:err:SXXP0003:Error reported by XML parser
    cause: The element type "link" must be terminated by the matching end-tag "</link>".

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':src_about.html'.
> XProc error err:XD0011

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
5 actionable tasks: 5 executed

C:\Data\Erik\work\xproc-3\xproc.org> 
```

**Comment by xatapult** [comment]
@ndw, things are weird (meaning I probably don't understand):

* I noticed several html files that were not in the repo in the `src` directory, so I removed all of that. I now have a clone that is exactly like the git repo again
* When I run gradle again it works! But... Leaves html files in the `src` directory, which are probably the cause of the failures in the previous comment?

So are the resulting html files written to the wrong directory maybe?
